src/interaction/interactiveTarget.js

```diff
@@ -88,18 +88,10 @@ export default {
     _pointerOver: false,
 
     /**
-     * Internal check to detect if the pointer is down on the displayObject
+     * Internal set of all active pointers, by identifier
      *
-     * @inner {boolean}
-     * @private
-     */
-    _pointerDown: false,
-
-    /**
-     * Internal check to detect the number of active touches on the displayObject
-     *
-     * @inner {number}
+     * @inner {Set}
      * @private
      */
-    _touchCount: 0,
+    _pointerIdentifiers: new Set(),
```

Comment:
Map and Set are both not supported on IE 9 & 10. Would suggest added a polyfill to fallback to Object.